application-security/admin_guide/get-started/connect-your-repositories/code-repositories/add-bitbucket.adoc

```diff
@@ -0,0 +1,87 @@
+
+. Configure a Bitbucket account on Prisma Cloud.
+
+.. Select *Settings > Code & Build Providers > Add > Bitbucket* (under Code Repositories).
++
+image::azure-repos-select.png[width=600]
+
+.. Select *Authorize* to authorize your Bitbucket account to integrate with Prisma Cloud.
++
+You are redirected to the Bitbucket login page.
+
+.. Enter you Bitbucket account details and then select *Continue*.
+
+.. Review the permissions and then select *Grant access* to confirm Prisma Cloud authorization on *Bitbucket*.
++
+You are redirected to the *Additional Permissions (Optional)* step of the wizard.
++
+image::bb-cicd-int.png[width=600]
++
+Optionally, you can enable the CI/CD Security module to gain visibility into, and protect your Bitbucket environment.
+
+.. Add your user name in the *User Name* field.
++
+NOTE: To retrieve your user name: In Bitbucket, select *Settings* (next to your profile) > *Personal Bitbucket settings*. Your user name is displayed under *Bitbucket profile settings*.
+
+. Generate an App Password.
+.. In Bitbucket, select *Settings* (next to your profile) > *Personal Bitbucket settings*.
+.. Select *App Passwords* under *Access Management* > *Create app password*.
+.. Provide a label and select the required permissions > *Create*.
++
+NOTE: You must grant Repositories Read and Admin permissions when creating the app password.
+.. Copy and save your password that is displayed and select *Close*.
+.. Enter the password in the *App Password* field of the wizard > *Next*.
+
+. Define the repositories to be scanned
+.. Select an available option:
++
+* *Permit all existing repositories*: Enables Prisma Cloud to scan all existing repositories that are associated with the selected app password
+* *Permit all existing and future repositories*: Enables Prisma Cloud to scan all existing repositories and any new repositories that are subsequently associated with the app password
+* *Choose from repository list*: This option enables you to select specific repositories to be scanned
+
+. Select *Next*.
+
+. Select *Done* in the *Status* step of the wizard that displays the *New integration successfully configured* message.
+
+. Verify that the Azure Repos integration with Prisma Cloud is successful:
+
+.. Select *Settings* > *Code & Build Providers*.
+
+. Verify that the *Bitbucket* integration is displayed from the  *VCS User Token* column.
++
+NOTE: You may have to wait for up to three minutes before the status of the integration is updated and displays
++
+image::azure-repos-9.1.png[width=800]
++
+After successfully completing integration, a new webhook is displayed in Bitbucket under *Settings* > *Workflow* > *Webhooks*.
++
+image::BitbucketWebhook.png[width=800]
```

Comment:
@JBakstPaloAlto - Please add this image to _graphics folder. Also the file name is always lower case.